pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -47,6 +47,11 @@ import (
 
 var bridgeFakeIP = "169.254.75.1%d/32"
 
+// the hardcoded MAC must be as low as possible, to prevent libvirt from
+// assigning a lower MAC to a tap device attached to the bridge - which
+// would trigger the bridge's MAC to update.
+const hardcodedMasqueradeMAC = "02:00:00:00:00:00"
```

Comment:
Please update the comment to include k6t-eth* nic which is also connected to the bridge.

Reply:
Done.